src/Controls/src/Core/BoxView.cs

```diff
@@ -34,9 +34,7 @@ public CornerRadius CornerRadius
 			return _platformConfigurationRegistry.Value.On<T>();
 		}
 
-		[Obsolete("OnSizeRequest is obsolete as of version 2.2.0. Please use OnMeasure instead.")]
-		[EditorBrowsable(EditorBrowsableState.Never)]
-		protected override SizeRequest OnSizeRequest(double widthConstraint, double heightConstraint)
+		protected override SizeRequest OnMeasure(double widthConstraint, double heightConstraint)
```

Comment:
@hartez is this correct? I know this has been obsolete for years, but has it really been obsolete obsolete?

Reply:
If not, now is it the time.